src/screens/hour.screen.tsx

```diff
@@ -16,12 +16,15 @@ import { WeatherData, WeatherDataDaySummary } from '../utils/weatherData';
 
 type ScreenProps = NativeStackScreenProps<RootDrawerParamList, 'Hourly'>;
 function HourScreen({ route, navigation }: ScreenProps): JSX.Element {
-  const { location, forecast, dayString, noValuesBefore, title } = route.params;
+  const { location: location_param, dayString, noValuesBefore, title } = route.params;
 
-  const { lat, lon } = useSelector((state: RootState) => state.location);
+  const { name: location_store, lat, lon } = useSelector((state: RootState) => state.location);
```

Comment:
maybe rename to `stored_location` or something like that? Will that be correct?

Reply:
Yes, that is better. Then it will be `store_location_name` and `param_location_name`. Sounds ok?

Reply:
Or should I rather keep `location_name` for param as kind of default, and then `store_location_name` for the second one?